consent/models.py

```diff
@@ -239,3 +246,150 @@ class Meta(Consent.Meta):
         help_text=_("Course key for which data sharing consent is granted.")
     )
     history = HistoricalRecords()
+
+
+@python_2_unicode_compatible
+class DataSharingConsentPage(TimeStampedModel):
+    """
+    Stores texts to be used in data sharing consent page.
+    """
+
+    class Meta(object):
+        app_label = 'consent'
+
+    TOP_PARAGRAPH_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for first paragraph of page. The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>platform_name: Name of platform.</li>'
+        '</ul>'
+    ))
+    POLICY_PARAGRAPH_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for policy paragraph at the bottom of page. The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>platform_name: Name of platform.</li>'
+        '</ul>'
+    ))
+    SIDEBAR_PARAGRAPH_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for left sidebar paragraph. The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>platform_name: Name of platform.</li>'
+        '</ul>'
+    ))
+    CONFIRMATION_MODAL_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for dialog which appears when user decline to provide consent. '
+        'The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>item: A string which is "course" or "program" depending on the type of consent.</li>'
+        '</ul>'
+    ))
+
+    page_title = models.CharField(
+        max_length=255,
+        default=_('Data sharing consent required'),
+        help_text=_('Title of page')
+    )
+    left_sidebar_header = models.CharField(
+        max_length=255,
+        default=_('Welcome to {platform_name}.'),
+        help_text=_('Header text in left sidebar. platform_name variable may be available')
+    )
+    left_sidebar_text = models.TextField(
+        null=True,
+        blank=True,
+        default=_(
+            "{enterprise_customer_name} has partnered with {platform_name} to offer you high-quality learning "
+            "opportunities from the world's best universities."
+        ),
+        help_text=SIDEBAR_PARAGRAPH_HELP_TEXT
+    )
+    message_header = models.TextField(
+        null=True,
+        blank=True,
+        default=_('Consent to share your data'),
+        help_text=_('Heading of page')
+    )
+    top_paragraph = models.TextField(
+        null=True,
+        blank=True,
+        default=_('First paragraph of page'),
+        help_text=TOP_PARAGRAPH_HELP_TEXT
+    )
```

Comment:
same comment about message_header and top_paragraph, maybe combine into one field?